app/main.development.js

```diff
@@ -30,7 +30,7 @@ const installExtensions = async () => {
       'REDUX_DEVTOOLS'
     ];
     const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
-    for (const name of extensions) {
+    for (const name of extensions) { // eslint-disable-line
```

Comment:
Airbnb restrict 'for ... of' statement in their style guide.
https://github.com/airbnb/javascript/pull/1134